core/drivers/crypto/caam/hal/common/hal_cfg.c

```diff
@@ -98,3 +110,12 @@ void __weak caam_hal_cfg_setup_nsjobring(struct caam_jrcfg *jrcfg)
 			caam_hal_jr_prepare_backup(jrcfg->base, jr_offset);
 	}
 }
+
+__weak void caam_hal_cfg_hab_jr_mgmt(struct caam_jrcfg *jrcfg __unused)
```

Comment:
Why are you using a weak function instead of configuration variables for this configuration?

Reply:
Sorry @jenswi-linaro, I don't understand your point.
Can you please give me a reference for this ?

Reply:
We normally do configuration using CFG_ variables unless it becomes too complicated.
In this case, I can't see what the problem would be, so I'm wondering if it was done for no particular reason.